src/kernel/src/fs/mod.rs

```diff
@@ -272,6 +273,27 @@ impl Fs {
         Ok(written.into())
     }
 
+    fn sys_writev(self: &Arc<Self>, i: &SysIn) -> Result<SysOut, SysErr> {
+        let fd: i32 = i.args[0].try_into().unwrap();
+        let ptr: *const libc::iovec = i.args[1].into();
+        let len: usize = i.args[2].try_into().unwrap();
+
+        let td = VThread::current().unwrap();
+        let file = td.proc().files().get(fd).ok_or(SysErr::Raw(EBADF))?;
```

Comment:
This also isn't that simple. I assume you got this from sys_write, but it's incomplete there. It might be ok for now though, we'll have to wait for ultima.

Reply:
@ultimaweapon What do you think about this?

Reply:
Do you mean this line? If yes I don't have any problem with it. But I did not take a look on the PS4 so I can't tell if it is correct.

Reply:
It should be fget_write, which does some additional checks and sets some flags. This one line is a 'weak' version of fget_write

Reply:
@SuchAFuriousDeath can you create an issue for this this so we don't forget?